index.html

```diff
@@ -153,7 +150,7 @@ <h2>
         <code>Badge</code> interface
       </h2>
       <pre class="idl">
-        [Exposed=(Window, Worker)]
```

Comment:
In this version of the API we are not supporting ServiceWorkers. I think having both will just cause confusion.

Reply:
sgtm